tensilelite/Tensile/Components/LSU.py

```diff
@@ -245,6 +242,12 @@ def writeReadReduction(self, writer, kernel):
                 DSLoadBX     = DSLoadB32
                 numInstPerVW = bytesPerVector // 4
                 regsPerStore = 1
+            maxOffset =  (kernel["LocalSplitU"] -1) * ldsStride + ((numVgprPerLSU // self.LSUfullVw -1) * (numInstPerVW-1) + numInstPerVW) * regsPerStore * (bpr * kernel["WavefrontSize"])
+            numAddr =maxOffset // maxLDSConstOffset + 1
+            if  writer.states.archCaps["Has160KLDS"]:
```

Comment:
Could we just use writer.vgprPool.checkOut(numAddr,"addr") only?
numAddr can be 1 so we don't need this if-else.

Reply:
For this PR, I think we can remove all if statements related to Has160KLDS.